tikv-client/src/main/java/com/pingcap/tikv/util/ConverterUpstream.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tikv.util;
+
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Encapsulate some the parameters that we maintain that are different from upstream into upstream
+ * library parameters to be consistent with the parameter types maintained everywhere. Upstream
+ * parameters are also converted to downstream parameters when needed. Distinguish convert/reConvert
+ * by name (downstream converts to upstream, upstream converts to downstream).
+ */
+public class ConverterUpstream {
```

Comment:
This is a configure transform util. Maybe another name is better.

Reply:
> This is a configure transform util. Maybe another name is better.

ConvertUpstreamUtils？